api/libproject/devstart.rb

```diff
@@ -744,22 +744,27 @@ def run
 })
 Common.register_command({
   :invocation => "generate-cdr-counts",
-  :description => "Generates databases in bigquery and data from a cdr that will be in cloudsql.",
+  :description => "generate-cdr-counts --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT> \
+--public-project <PROJECT> --account <ACCOUNT> --cdr-version=YYYYMMDD --bucket <BUCKET>
+Generates databases in bigquery with data from a cdr that will be imported to mysql/cloudsql to be used by workbench and databrowser.",
   :fn => lambda { |*args| generate_cdr_counts(*args) }
 })
 Common.register_command({
   :invocation => "generate-local-cdr-db",
-  :description => "Creates and populates mysql or cloudsql database from data made by generate-cdr-counts.",
+  :description => "generate-cloudsql-cdr --account <ACCOUNT> --cdr-version <''|YYYYMMDD> --cdr-db-prefix <cdr|public> --bucket <BUCKET>
```

Comment:
If cdr-version is optional, better to put:

[--cdr-version YYYYMMDD]

Reply:
Ping.

Reply:
It's not optional. It has to be '' .  That could be a todo . I had trouble making an option optional